src/Microsoft.AspNet.Mvc.Core/ModelBinding/BodyModelBinder.cs

```diff
@@ -64,9 +64,9 @@ public BodyModelBinder()
                     // system to skip other model binders and never to fall back.
                     return new ModelBindingResult(modelBindingKey);
                 }
-
-                var valueProviderResult = new ValueProviderResult(rawValue: model);
-                bindingContext.ModelState.SetModelValue(modelBindingKey, valueProviderResult);
+                
+                // We don't have anything useful to put in model state.
```

Comment:
> We wanted to consistently make the bound values available in the ModelStateDictionary. Without either "overloading" RawValue or adding that new ModelState property, looks like a regression.

But we never did that... `RawValue` had the values from the value provider in most cases, and the value created by the model binder in the case of binding `IFormFile` or `[FromBody]`. This is definitely a change and I think it's in the right direction.


>Note users can write into ModelStateDictionary. Also HTML generation converts whatever is in RawValue to string and the few other types it needs.

Yes. This is also intentionally changed to be not supported with arbitrary objects. If you want to do that, then you're responsible for `ToString()`ing the object, which seems pretty reasonable.